bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/ThingManager.java

```diff
@@ -162,8 +162,12 @@ public void statusUpdated(final Thing thing, ThingStatusInfo thingStatus) {
                 Bridge bridge = (Bridge) thing;
                 for (Thing bridgeThing : bridge.getThings()) {
                     if (thingStatus.getStatus() == ThingStatus.ONLINE) {
-                        ThingStatusInfo statusInfo = ThingStatusInfoBuilder.create(ThingStatus.ONLINE).build();
-                        setThingStatus(bridgeThing, statusInfo);
+                    	ThingStatusInfo bridgeThingStatus = bridgeThing.getStatusInfo(); 
+                    	if (bridgeThingStatus.getStatus() == ThingStatus.ONLINE || 
```

Comment:
OK, I see - but there is a difference: You refer to a comment which applies for a binding setting a status. The information when the binding had the last successful communication with a thing might be helpful. 

Here the framework updates the status, so I think it is not needed to set ONLINE again, if the binding already set it to ONLINE before. Better to remove the check, as it might be a bit confusing.